### PR TITLE
nu@0.8.0: Fix extract_dir

### LINKED ISSUE #718: Incorrect extract_dir for nu 0.8.0
`"extract_dir": "nu_0_7_0_windows\\nushell_0_7_0"`
The `extract_dir` should be:
`"extract_dir": "nu_0_8_0_windows\\nushell_0_8_0"`

### PR DESCRIPTION
**Pull request by myty**
Closes #718

### CODE REVIEW

**Comment by linsui** [comment]
The autoupdate didn't work properly. Please also fix that.

```
{
    "homepage": "https://www.nushell.sh",
    "description": "A modern shell written in Rust",
    "version": "0.8.0",
    "license": "MIT",
    "architecture": {
        "64bit": {
            "url": "https://github.com/nushell/nushell/releases/download/0.8.0/nu_0_8_0_windows.zip",
            "hash": "7e34ede344f979130517ae5d2507c71aac449fb67aa4601b09bb252a41efa95b"
        }
    },
    "extract_dir": "nu_0_8_0_windows\\nushell_0_8_0",
    "installer": {
        "script": [
            "New-Item \"$dir\\Plugins\" -ItemType Directory | Out-Null",
            "Move-Item \"$dir\\nu_plugin_*\" \"$dir\\Plugins\""
        ]
    },
    "bin": "nu.exe",
    "env_add_path": "Plugins",
    "checkver": {
        "github": "https://github.com/nushell/nushell",
        "regex": "([\\d.]+) Release</a>"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/nushell/nushell/releases/download/$version/nu_$underscoreVersion_windows.zip"
            }
        },
        "extract_dir": "nu_$underscoreVersion_windows\\nushell_$underscoreVersion"
    }
}

```

**Comment by myty** [comment]
> The autoupdate didn't work properly. Please also fix that.

Will do.

**Comment by myty** [comment]
I think autoupdate didn't work because of the extract_dir being incorrect.  I ran `.\bin\checkver.ps1 nu -u` and nothing changed.

**Comment by niheaven** [comment]
```json
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/nushell/nushell/releases/download/$version/nu_$underscoreVersion_windows.zip",
        		"extract_dir": "nu_$underscoreVersion_windows\\nushell_$underscoreVersion"
            }
        }
    }
```

Move `extract_dir` into arch

File: bucket/nu.json [hunk 1]
```diff
@@ -23,7 +23,11 @@
         "regex": "([\\d.]+) Release</a>"
     },
     "autoupdate": {
-        "url": "https://github.com/nushell/nushell/releases/download/$version/nu_$underscoreVersion_windows.zip",
-        "extract_dir": "nu_$underscoreVersion_windows\\nushell_$underscoreVersion"
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/nushell/nushell/releases/download/$version/nu_$underscoreVersion_windows.zip",
+        		"extract_dir": "nu_$underscoreVersion_windows\\nushell_$underscoreVersion"
```
Comment (nightscape) [inline bucket/nu.json]: Formatting

**Pull request merged**